arm-sql/2015-05-01/swagger/2015-05-01-all.json

```diff
@@ -0,0 +1,1914 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
```

Comment:
there are few other issues as well which should be addressed in the newer version of API, we can talk about the same in coming meeting

---
In reply to: [84132972](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r84132972) [](ancestors = 84132972)